package.json

```diff
@@ -1,40 +1,38 @@
 {
     "name": "docs-blockchain",
-    "workspaces": [
-        "packages/*"
-    ],
+    "type": "module",
     "scripts": {
-        "cleanup": "npx ts-node ./scripts/cleanup.ts",
-        "update:files": "npx ts-node ./scripts/cascadeFiles.ts",
+        "[-] Main Script for Realtime Updates": "",
         "dev": "vitepress dev docs",
-        "dev:staging": "npm run update:files && vitepress dev ./packages/staging/docs",
-        "dev:mainnet": "npm run update:files && vitepress dev ./packages/mainnet/docs",
-        "build:experimental": "vitepress build docs",
-        "build:staging": "vitepress build ./packages/staging/docs",
-        "build:mainnet": "vitepress build ./packages/mainnet/docs",
-        "build:together": "npm run build:mainnet & npm run build:staging & npm run build:experimental",
-        "build:all": "npm run update:files && npm run build:together && npx ts-node ./scripts/reorganize.ts && npx ts-node ./scripts/sitemap.ts"
+        "[-] Run Development Versions": "",
+        "dev:experimental": "npx tsx ./scripts/scaffold.ts && vitepress dev ./docs-experimental",
+        "dev:staging": "npx tsx ./scripts/scaffold.ts && vitepress dev ./docs-staging",
+        "dev:mainnet": "npx tsx ./scripts/scaffold.ts && vitepress dev ./docs-mainnet",
+        "[-] Build Distributable Files - Make sure to scaffold files first if ran individually": "",
+        "build:experimental": "vitepress build ./docs-experimental",
+        "build:staging": "vitepress build ./docs-staging",
+        "build:mainnet": "vitepress build ./docs-mainnet",
+        "build:together": "npx tsx ./scripts/scaffold.ts && npm run build:mainnet & npm run build:staging & npm run build:experimental",
+        "build:all": "npm run build:together && npx tsx ./scripts/reorganize.ts && npx tsx ./scripts/sitemap.ts",
+        "[-] Preview Scaffolded Files ": "",
+        "preview:experimental": "tsx ./scripts/scaffold.ts && npm run build:experimental && vitepress preview ./docs-experimental",
```

Comment:
What is the point of scaffolding if you can run development version? And if you want the page files can't you just build?